resources/views/inputs/upload-image-single.blade.php

```diff
@@ -42,18 +43,29 @@
         {{ $minHeight }},
         {{ $maxWidth }},
         {{ $maxHeight }},
-        {{ $width }},
-        {{ $height }},
+        '{{ null }}',
+        '{{ null }}',
```

Comment:
This will result in an empty string, and it expects  `null` or `number`

```suggestion
        null,
        null,
```

Reply:
@alfonsobries ops, forgot to change!